src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -8,7 +8,7 @@ export default class InternalPushProvider implements W3iPushProvider {
   private readonly emitter: EventEmitter
   public providerName = 'InternalChatProvider'
 
-  public constructor(emitter: EventEmitter) {
+  public constructor(emitter: EventEmitter, _name = 'internal') {
```

Comment:
Guess I'm missing something but I'm actually not clear on the `_` prefix here, why not just do default assignment as `name = 'internal'`?

Reply:
Simply to denote it's not used anywhere